### ISSUE TITLE
Expected libstdc++ in Trusty .deb is newer than what's included in Trusty

### ISSUE THREAD

**Issue by lae** [nicehash/excavator#88]
The Trusty .deb release does not install on a Ubuntu Trusty installation:

```
~$ sudo dpkg -i excavator_1.3.8a-trusty0_amd64.deb 
(Reading database ... 124409 files and directories currently installed.)
Preparing to unpack excavator_1.3.8a-trusty0_amd64.deb ...
Unpacking excavator (1.3.8a-trusty0) over (1.3.8a-trusty0) ...
dpkg: dependency problems prevent configuration of excavator:
 excavator depends on libstdc++6 (>= 5); however:
  Version of libstdc++6:amd64 on system is 4.8.4-2ubuntu1~14.04.3.

dpkg: error processing package excavator (--install):
 dependency problems - leaving unconfigured
Processing triggers for mime-support (3.54ubuntu1.1) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Errors were encountered while processing:
 excavator
```

It looks like it's expecting a newer version of libstdc++ but that does not exist in the Trusty repositories?

```
~$ apt-cache showpkg libstdc++
Package: libstdc++6
Versions: 
4.8.4-2ubuntu1~14.04.3 (/var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty-updates_main_binary-amd64_Packages) (/var/lib/apt/lists/security.ubuntu.com_ubuntu_dists_trusty-security_main_binary-amd64_Packages) (/var/lib/dpkg/status)
 Description Language: 
                 File: /var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty_main_binary-amd64_Packages
                  MD5: 724ab84919e0e220afb960e36463914d
 Description Language: en
                 File: /var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty_main_i18n_Translation-en
                  MD5: 724ab84919e0e220afb960e36463914d

4.8.2-19ubuntu1 (/var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty_main_binary-amd64_Packages)
 Description Language: 
                 File: /var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty_main_binary-amd64_Packages
                  MD5: 724ab84919e0e220afb960e36463914d
 Description Language: en
                 File: /var/lib/apt/lists/us.archive.ubuntu.com_ubuntu_dists_trusty_main_i18n_Translation-en
                  MD5: 724ab84919e0e220afb960e36463914d
```

**Comment by dropky** [comment]
We'll look into this and fix it in the next release. I will keep you updated.

**Comment by dropky** [comment]
This should be solved with the v1.3.9 release. If not, please let me know.
